### PR TITLE
easy_thresh: Add missing imports; Fix regex escaping

### PR DESCRIPTION
**Pull request by nx10**
## Description

Add missing imports to easy_thresh module and make an unescaped regex string literal raw.
I wonder how this was able to run before.

## Checklist

- [x] My pull request has a descriptive title (not a vague title like `Update index.md`).
- [x] My pull request targets the `develop` branch of the repository. 
- [x] My commit messages follow best practices.
- [x] My code follows the established code style of the repository.
- [ ] I added tests for the changes I made (if applicable).
- [ ] I updated the changelog.
- [ ] I added or updated documentation (if applicable).
- [x] I tried running the project locally and verified that there are no visible errors.
  - I ran the tests locally in docker

## Developer Certificate of Origin

<details>
<summary>Developer Certificate of Origin</summary>

```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>

### CODE REVIEW

**Review by shnizzedy — CHANGES_REQUESTED** [review]
> ["First-time contributor"](https://github.com/FCP-INDI/C-PAC/pull/1846#issue-1453691797)

:clap: :tada: 

> Add missing imports to easy_thresh module […].
> I wonder how this was able to run before.

Nipype has some unpythonic structure, and one that took me a lot of getting used to is

> If you want to use another module inside a function, you have to import it again inside the function.

― [Important - Function Nodes are closed environments](https://miykael.github.io/nipype_tutorial/notebooks/basic_function_interface.html#Important---Function-Nodes-are-closed-environments)

In [C-PAC's variant](https://github.com/FCP-INDI/C-PAC/blob/9586d7146eff224200e18622523a56da6a9826a6/CPAC/utils/interfaces/function/function.py#L14-L170) of [Nipype's Function Node Interface](https://github.com/nipy/nipype/blob/071523a5069b52689ed27f995f1e2bea5b6e0402/nipype/interfaces/utility/wrappers.py#L24-L159), we have the option to be a little more pythonic by passing https://github.com/FCP-INDI/C-PAC/blob/9586d7146eff224200e18622523a56da6a9826a6/CPAC/utils/interfaces/function/function.py#L58-L60

> [M]ake an unescaped regex string literal raw.

:sunglasses:

File: CPAC/easy_thresh/easy_thresh.py [hunk 1]
```diff
@@ -372,7 +376,7 @@ def call_cluster(in_file, volume, dlh, threshold, pthreshold, parameters):
 
     f = open(localmax_txt_file,'wb')
 
-    cmd = sb.Popen([ cmd_path,
+    cmd = subprocess.Popen([ cmd_path,
```
[the remark this answers is not in the document]
Comment (nx10) [reply CPAC/easy_thresh/easy_thresh.py:379]: Fixed with https://github.com/FCP-INDI/C-PAC/blob/903d2e8e8917f77a3b33b124a40b9c422bf457b6/CPAC/easy_thresh/easy_thresh.py#L239

File: CPAC/easy_thresh/easy_thresh.py [hunk 2]
```diff
@@ -451,7 +455,7 @@ def get_standard_background_img(in_file, file_parameters):
 
     try:
 
-        img = load(in_file)
+        img = nib.load(in_file)
```
[the remark this answers is not in the document]
Comment (nx10) [reply CPAC/easy_thresh/easy_thresh.py:458]: Fixed with https://github.com/FCP-INDI/C-PAC/blob/903d2e8e8917f77a3b33b124a40b9c422bf457b6/CPAC/easy_thresh/easy_thresh.py#L286

**Comment by nx10** [comment]
Thank you for the quick response and thorough explanations!

I am guessing nipype restricts global scope access to avoid dealing with most of the python interpreter lock headache.

**Review by shnizzedy — APPROVED** [review]

**Review by sgiavasis — APPROVED** [review]

**Pull request merged**